projects/eHealthSuisse/eMedikation/v1.0/1-1-MedicationTreatmentPlan.xml

```diff
@@ -134,28 +103,15 @@
 			</informationRecipient>
 		</intendedRecipient>
 	</informationRecipient>
-
-	<!-- closed not yet available <legalAuthenticator> <time value="20111129110000+0100" 
-		/> <signatureCode code="S" /> <assignedEntity> <id nullFlavor="NI" /> <assignedPerson> 
-		<name> <family>Hausarzt</family> </name> </assignedPerson> </assignedEntity> 
-		</legalAuthenticator> -->
-	<component contextConductionInd='true'>
+	<component contextConductionInd="true">
 		<structuredBody>
 			<component>
 				<section>
-
-					<!-- CH-PHARM MTP -->
-					<templateId root='2.16.756.5.30.1.1.10.3.13' />
-					<!-- IHE PHARM MTP -->
-					<templateId root='1.3.6.1.4.1.19376.1.9.1.2.6' />
-
+					<templateId root="2.16.756.5.30.1.1.10.3.13" />
+					<templateId root="1.3.6.1.4.1.19376.1.9.1.2.6" />
 					<id root="C9F758A1-296C-4710-84D4-E181DB8C7478" />
-
-					<code code="77604-7" codeSystem="2.16.840.1.113883.6.1"
-						codeSystemName="LOINC" displayName="Medication treatment plan.brief" />
-
-					<title>Medikamentöser Behandlungsplan</title>
-
+					<code code="77604-7" codeSystem="2.16.840.1.113883.6.1" codeSystemName="LOINC" displayName="Medication treatment plan.brief" />
+					<title>Medikamentöser Behandlungsplan</title>

```

Comment:
That's a vicious one: the "ö" is encoded as LATIN SMALL LETTER O + COMBINING DIAERESIS (two code points) instead of LATIN SMALL LETTER O WITH DIAERESIS (single code point, the ASCII one you should get with your keyboard). The schematron assert had to be dropped to allow other languages, it would have caught it otherwise